docs/source/index.rst

```diff
@@ -62,7 +62,15 @@ Conduct`_.
 Licence
 -------
 
-Copyright © 2022-2024 Met Office and contributors. CSET is distributed under the
-`Apache 2.0 License`_, in the hope that it will be useful.
+© Crown copyright, Met Office (2022-2024) and CSET contributors.
 
-.. _Apache 2.0 License: https://www.apache.org/licenses/LICENSE-2.0
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You may obtain a
+copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
```

Comment:
I know this was all caps in the email but do you think it should be lowercase after the "AS IS"?

Reply:
It is capitalised that way in the example given on the [Apache 2.0 licence page](https://www.apache.org/licenses/LICENSE-2.0#apply) so is probably correct. Furthermore it is fairly common for the warranty bit to be capitalised, such as in the MIT licence or BSD licence.